src/main/resources/db/migration/V5.13.0.4__8092-timestamp-of-data-access-request.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE fileaccessrequests
+ADD COLUMN IF NOT EXISTS creation_time TIMESTAMP WITHOUT TIME ZONE DEFAULT now();
```

Comment:
That was indeed the problem I faced while building this. If the migration scripts support queries as well, we could do some kind of group by trick and table renaming to remove duplicate rows and create a fresh table with only unique rows. 

This will also require some changes in the code however. 

So the question is, would something like this:

```sql
-- note this is pseudo-sql
create table tmp_fileaccessrequests (user_id, file_id);

insert into tmp_fileaccessrequests 
select distinct user_id, file_id from fileaccessrequests;

drop table fileaccessrequests;
rename table tmp_fileaccessrequests to fileaccessrequests;
```

be an acceptable solution?